### ISSUE TITLE
Some ReliabilityProblems have too many parameters.

### ISSUE THREAD

**Issue by mbaudin47** [mbaudin47/otbenchmark#24]
Some ReliabilityProblems have too many optional parameters.

For example, the ReliabilityProblem60 has 11 optional parameters:

https://github.com/mbaudin47/otbenchmark/blob/7ca5d75e1f2b0232fa7885f3ac1bb3d21784050b/otbenchmark/ReliabilityProblem60.py#L15

This is the same for other problems as well:

https://github.com/mbaudin47/otbenchmark/blob/master/otbenchmark/ReliabilityProblem110.py
https://github.com/mbaudin47/otbenchmark/blob/master/otbenchmark/ReliabilityProblem111.py

etc...

In the case where there are several random inputs with the same parameters, we should take a list of mus and sigmas instead of individual mus and sigma. Hence, the new API would be:

```
ReliabilityProblem60(threshold, mu, sigma)
```

where `mu` and `sigma` are lists of length 5.

@JEBROUN : do you agree on this point?

A first step would be to make the list of all the problems in this situation.

**Comment by JEBROUN** [comment]
Yes, I am agree with you, therefore, I check if the parameters of a problem are the same, I modify the API and also some lines of code.

**Comment by JEBROUN** [comment]
you want me to make a list of all the problems or just a list of the problems which contains the same parameters ???

**Comment by mbaudin47** [comment]
Neither. Some constructors have to many input arguments, because they have several mu1, mu2, ..., mu5 and several sigma1, sigma2, ...., sigma5. In this case, just two input arguments are sufficient : mu = [mu1, mu2, ...., mu5] and sigma = [sigma1, sigma2, ..., sigma5]. The drawback is that, we cannot customize e.g. mu3 alone: the full list of mu* must be configured. 
For example, the constructor of ReliabilityProblem110.py:
```
def __init__(self, threshold=0.0, mu1=0.0, sigma1=1.0, mu2=0.0, sigma2=1.0):
```
would change to:
```
def __init__(self, threshold=0.0, mu=[0.0]*2, sigma=[1.0]*2):
```
This is simpler, shorter and more readable.

**Comment by mbaudin47** [comment]
This bug is fixed thanks to @JEBROUN at 1d7b66df5f6a2ebc86d21e04554bc8ecdb5076cd. This is why I close this ticket.
